src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -8071,47 +8089,80 @@ void Player_Action_808407CC(Player* this, PlayState* play) {
     }
 }
 
-void func_808409CC(PlayState* play, Player* this) {
+// Get the offset of a common special idle type relative to the other common types
```

Comment:
"common special" sounds like an oxymoron. Maybe these are truly the "non-environmental" ones?

Reply:
would "fidget" as suggested fix this problem for you? They are common in that they can still be used even in environmental contexts (depending on rng)

Reply:
yeah "fidget" would resolve this for me